add-on/panels/js/roomToc.jsx

```diff
@@ -12,51 +13,71 @@ loop.roomToc = (function() {
   function init() {
     loop.shared.utils.getBoolPreference = function foo() {};
 
-    var dispatcher = new loop.Dispatcher();
-    var sdkDriver = new loop.OTSdkDriver({
-      // For the standalone, always request data channels. If they aren't
-      // implemented on the client, there won't be a similar message to us, and
-      // we won't display the UI.
-      constants: {},
-      useDataChannels: true,
-      dispatcher: dispatcher,
-      sdk: OT
-    });
+    var requests = [
+      ["GetAllConstants"],
+      ["GetAllStrings"],
+      ["GetLocale"]
+    ];
 
-    var activeRoomStore = new loop.store.ActiveRoomStore(dispatcher, {
-      sdkDriver: sdkDriver
-    });
+    loop.requestMulti.apply(null, requests).then(function(results) {
+      let [constants, stringBundle, locale] = results;
+      mozL10n.initialize({
+        locale: locale,
+        getStrings: function(key) {
+          if (!(key in stringBundle)) {
+            console.error("No string found for key: ", key);
+            return JSON.stringify({ textContent: "foo" });
+          }
 
-    // XXX akita: Get loop constants
-    var roomStore = new loop.store.RoomStore(dispatcher, {
-      constants: { CONSTANT: false }
-    });
+          return JSON.stringify({ textContent: stringBundle[key] });
+        }
+      });
 
-    loop.store.StoreMixin.register({
-      activeRoomStore: activeRoomStore,
-      roomStore: roomStore
-    });
+      var dispatcher = new loop.Dispatcher();
+      var sdkDriver = new loop.OTSdkDriver({
+        // For the standalone, always request data channels. If they aren't
+        // implemented on the client, there won't be a similar message to us, and
+        // we won't display the UI.
+        constants: {},
```

Comment:
Looks like most other places pass in constants, although I didn't really look at what it's being used for..